File: Modul 5/Soal 5/PRAK505-2410817210014-ZAVIERGHANI.c
#include <stdio.h>

void Biodata(int tahunLahir, char Nama[], char Asal[]) {
    int tahun_sekarang = 2020;
    int umur = tahun_sekarang - tahunLahir;

    printf("Perkenalkan Nama Saya : %s\n", Nama);
    printf("Umur Saya : %d\n", umur);
    printf("Saya Adalah Angkatan : %d\n", tahun_sekarang);
    printf("Asal Saya dari : %s\n", Asal);
}

int main() {
    int tahunLahir;
    char A[20], B[15];

    printf("Masukkan tahun lahir: ");
    scanf("%d", &tahunLahir);
    getchar(); 
    
    printf("Masukkan nama: ");
    scanf("%[^\n]%*c", A); 
    
    printf("Masukkan asal: ");
    scanf("%[^\n]%*c", B); 

    Biodata(tahunLahir, A, B);
    
    return 0;
}

File: Modul 5/Soal 5/PRAK505-2410817210014-ZAVIER GHANI.py
def Biodata(tahunLahir, Nama, Asal):
    tahun_sekarang = 2020
    umur = tahun_sekarang - tahunLahir
    
    print(f"Perkenalkan Nama Saya : {Nama}")
    print(f"Umur Saya : {umur}")
    print(f"Saya Adalah Angkatan : {tahun_sekarang}")
    print(f"Asal Saya dari : {Asal}")

tahunLahir = int(input("Masukkan tahun lahir: "))
Nama = input("Masukkan nama: ")
Asal = input("Masukkan asal: ")

Biodata(tahunLahir, Nama, Asal)
